src/compute-shortname.js

```diff
@@ -56,6 +56,18 @@ function computeShortname(url) {
         return whatwg[1];
     }
 
+    // Handle ECMAScript
+    if (url === "https://tc39.es/ecma262/") {
+      return "ecmascript";
+    }
```

Comment:
Given that this only deals with one spec, could that appear in a `shortname` property in `specs.json` instead?